src/core.ts

```diff
@@ -589,12 +601,36 @@ export function stressSolution(path: string, times: number) {
     let solCompileResult = compileCode(sol, out);
     if (solCompileResult.status !== 0) {
         vscode.window.showErrorMessage(`Compilation Error. ${sol}`);
+        let error_path = join(path, "stderr");
+        let error_file = openSync(error_path, "w");
+        writeSync(error_file, solCompileResult.stderr.toString());
+        vscode.commands.executeCommand("vscode.setEditorLayout", {
+            orientation: 1,
+            groups: [{ groups: [{}, {}], size: 0.5 }],
```

Comment:
Thanks for this change @Salil03, it is great. I tested locally and it works nice, but I have a suggestion. 

Compilation errors are usual long errors, I think is better to split editor horizontally and show error in the bottom rather than splitting editor vertically.